### DISCUSSION TITLE
HUGO  THIS WAS AN ERROR WHILE SAVING THE ENTRY

### DISCUSSION THREAD

**Discussion by Tom80ok** [sveltia/sveltia-cms#254] [Q&A]
hello
As you can see i have this problem....

**Comment by kyoshino** [comment]
What message do you see in the browser console?

**Comment by Tom80ok** [comment]
Hello André thanks you for answer

I have This
THIS WAS AN ERROR WHILE SAVING THE ENTRY

When I want to changé a text or add an asset

Sorry for my english !

Le lun. 11 nov. 2024, 22:00, Kohei Yoshino ***@***.***> a
écrit :

**Comment by kyoshino** [comment]
I can’t help you unless you provide an error message in the browser console. If you’re not a developer, please ask your CMS administrator for help.
